traits/ctraits.c

```diff
@@ -901,10 +900,6 @@ has_traits_init ( PyObject * obj, PyObject * args, PyObject * kwds ) {
 
     PyObject * key;
     PyObject * value;
-    PyObject * klass;
-    PyObject * handler;
-    PyObject * handler_args;
-    int n;
     int has_listeners;
     Py_ssize_t i = 0;
```

Comment:
It looks like `i` isn't used anywhere in `has_traits_init` (now that the trait monitor code is out).

Reply:
never mind. im wrong. it is being used when setting traits.